--- fluxbox-0.9.14/data/Makefile.am.orig	2005-05-27 13:03:52.000000000 +0200
+++ fluxbox-0.9.14/data/Makefile.am	2006-02-08 13:05:13.000000000 +0100
@@ -8,13 +8,10 @@
 SUBDIRS= 		styles
 CLEANFILES= 		menu menu.in menu.pre menu.bak init
 MAINTAINERCLEANFILES= 	Makefile.in
-pkgdata_DATA= 		keys menu init
+pkgdata_DATA= 		keys init
 EXTRA_DIST= 		keys init.in README.style README.menu README
 all-local: 		menu init
 install-pkgdataDATA:
-	@echo "Installing menu file in $(DESTDIR)$(DEFAULT_MENU)"
-	$(mkinstalldirs) $(DESTDIR)$(dir $(DEFAULT_MENU))
-	$(INSTALL_DATA) menu $(DESTDIR)$(DEFAULT_MENU)
 	@echo "Installing keys file in $(DESTDIR)$(DEFAULT_KEYS)"
 	$(mkinstalldirs) $(DESTDIR)$(dir $(DEFAULT_KEYS))
 	$(INSTALL_DATA) keys $(DESTDIR)$(DEFAULT_KEYS)
@@ -29,11 +26,6 @@
 			-e "s,@default_style@,$(DEFAULT_STYLE),g" $(srcdir)/init.in > init
 
 menu: 			../util/fluxbox-generate_menu
-	env	MENUFILENAME=menu \
-		PREFIX="${prefix}" \
-		MENUTITLE="Fluxbox-${VERSION}" \
-		INSTALL=Yes \
-		../util/fluxbox-generate_menu
 
 ../util/fluxbox-generate_menu:
 	cd ../util && ${MAKE} fluxbox-generate_menu
