--- fluxbox-1.0.0/data/Makefile.am	2007-10-09 20:07:50.211422282 +0200
+++ fluxbox-1.0.0/data/Makefile.am-new	2007-10-09 20:10:10.427236759 +0200
@@ -11,13 +11,10 @@
 SUBDIRS= 		styles
 CLEANFILES= 		menu menu.in menu.pre menu.bak init
 MAINTAINERCLEANFILES= 	Makefile.in
-pkgdata_DATA= 		keys menu init
+pkgdata_DATA= 		keys init
 EXTRA_DIST= 		keys init.in
 all-local: 		menu init
 install-pkgdataDATA:
-	@echo "Installing menu file in $(DESTDIR)$(DEFAULT_MENU)"
-	$(mkinstalldirs) $(DESTDIR)$(dir $(DEFAULT_MENU))
-	$(INSTALL_DATA) menu $(DESTDIR)$(DEFAULT_MENU)
 	@echo "Installing keys file in $(DESTDIR)$(DEFAULT_KEYS)"
 	$(mkinstalldirs) $(DESTDIR)$(dir $(DEFAULT_KEYS))
 	$(INSTALL_DATA) $(srcdir)/keys $(DESTDIR)$(DEFAULT_KEYS)
@@ -34,12 +31,6 @@
 			-e "s,@default_style@,$(DEFAULT_STYLE),g" $(srcdir)/init.in > init
 
 menu: 			../util/fluxbox-generate_menu
-	env	MENUFILENAME=menu \
-		PREFIX="${prefix}" \
-		MENUTITLE="Fluxbox-${VERSION}" \
-		INSTALL=Yes \
-		PATH="${PATH}:../util:../util/fbrun" \
-		../util/fluxbox-generate_menu
 
 ../util/fluxbox-generate_menu:
 	cd ../util && ${MAKE} fluxbox-generate_menu
