diff -rNu fluxbox-0.9.14/data/styles.orig/Artwiz fluxbox-0.9.14/data/styles/Artwiz
--- fluxbox-0.9.14/data/styles.orig/Artwiz	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Artwiz	2006-02-08 13:44:01.000000000 +0100
@@ -104,7 +104,7 @@
 borderWidth:			1
 handleWidth:			5
 
-rootCommand:	                bsetroot -mod 4 4 -bg rgb:10/18/20 -fg rgb:30/38/40
+rootCommand:	                fbsetroot -mod 4 4 -bg rgb:10/18/20 -fg rgb:30/38/40
 
 *Font:	                        -*-lucidatypewriter-medium-r-*-*-*-100-*-*-*-*-*-*
 !*Font:                         vera-7
diff -rNu fluxbox-0.9.14/data/styles.orig/BlueNight fluxbox-0.9.14/data/styles/BlueNight
--- fluxbox-0.9.14/data/styles.orig/BlueNight	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/BlueNight	2006-02-08 13:44:01.000000000 +0100
@@ -5,7 +5,7 @@
 style.comments:                      Modified/tweaked by skypher
 
 
-rootCommand:                         bsetroot -solid rgb:6A/9A/AF
+rootCommand:                         fbsetroot -solid rgb:6A/9A/AF
 
 ! Toolbar settings...
 toolbar.button:                      Gradient Diagonal
diff -rNu fluxbox-0.9.14/data/styles.orig/Carbondioxide fluxbox-0.9.14/data/styles/Carbondioxide
--- fluxbox-0.9.14/data/styles.orig/Carbondioxide	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Carbondioxide	2006-02-08 13:44:01.000000000 +0100
@@ -132,4 +132,4 @@
 borderWidth:                         1
 handleWidth:                         3
 
-rootCommand:                         bsetroot -gradient flatverticalgradient -from black -to white
+rootCommand:                         fbsetroot -gradient flatverticalgradient -from black -to white
diff -rNu fluxbox-0.9.14/data/styles.orig/Clean fluxbox-0.9.14/data/styles/Clean
--- fluxbox-0.9.14/data/styles.orig/Clean	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Clean	2006-02-08 13:44:01.000000000 +0100
@@ -114,4 +114,4 @@
 borderWidth:                         1
 handleWidth:                         2
 
-rootCommand:                         bsetroot -solid rgb:9A/9A/90
+rootCommand:                         fbsetroot -solid rgb:9A/9A/90
diff -rNu fluxbox-0.9.14/data/styles.orig/Cthulhain fluxbox-0.9.14/data/styles/Cthulhain
--- fluxbox-0.9.14/data/styles.orig/Cthulhain	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Cthulhain	2006-02-08 13:44:01.000000000 +0100
@@ -117,7 +117,7 @@
 handleWidth:				4
 frameWidth:				0
 
-rootCommand:				bsetroot -solid "#3a404b"
+rootCommand:				fbsetroot -solid "#3a404b"
 
 ! ***** bbpager *****
 bbpager.frame:				sunken gradient crossdiagonal
diff -rNu fluxbox-0.9.14/data/styles.orig/Flux fluxbox-0.9.14/data/styles/Flux
--- fluxbox-0.9.14/data/styles.orig/Flux	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Flux	2006-02-08 13:44:01.000000000 +0100
@@ -88,4 +88,4 @@
 *textColor:			grey20
 *Font:				lucidasans-10
 
-rootCommand:			bsetroot -solid rgb:4/4/38
+rootCommand:			fbsetroot -solid rgb:4/4/38
diff -rNu fluxbox-0.9.14/data/styles.orig/LemonSpace fluxbox-0.9.14/data/styles/LemonSpace
--- fluxbox-0.9.14/data/styles.orig/LemonSpace	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/LemonSpace	2006-02-08 13:44:01.000000000 +0100
@@ -125,4 +125,4 @@
 window.tab.label.focus.textColor:    rgb:1a/04/08
 ! -
 
-rootCommand:                         bsetroot -solid rgb:db/bc/83
+rootCommand:                         fbsetroot -solid rgb:db/bc/83
diff -rNu fluxbox-0.9.14/data/styles.orig/Makro fluxbox-0.9.14/data/styles/Makro
--- fluxbox-0.9.14/data/styles.orig/Makro	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Makro	2006-02-08 13:44:01.000000000 +0100
@@ -113,4 +113,4 @@
 borderWidth:                         1
 handleWidth:                         3
 
-rootCommand:                         bsetroot -solid rgb:9A/9A/90
+rootCommand:                         fbsetroot -solid rgb:9A/9A/90
diff -rNu fluxbox-0.9.14/data/styles.orig/MerleyKay fluxbox-0.9.14/data/styles/MerleyKay
--- fluxbox-0.9.14/data/styles.orig/MerleyKay	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/MerleyKay	2006-02-08 13:44:01.000000000 +0100
@@ -131,4 +131,4 @@
 borderWidth:                         2
 handleWidth:                         3
 
-rootCommand:                         bsetroot -mod 5 5 -fg rgb:62/70/85 -bg rgb:52/60/75
+rootCommand:                         fbsetroot -mod 5 5 -fg rgb:62/70/85 -bg rgb:52/60/75
diff -rNu fluxbox-0.9.14/data/styles.orig/Meta fluxbox-0.9.14/data/styles/Meta
--- fluxbox-0.9.14/data/styles.orig/Meta	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Meta	2006-02-08 13:44:01.000000000 +0100
@@ -124,5 +124,5 @@
 frameWidth:                          1
 handleWidth:                         4
 
-rootCommand:                         bsetroot -solid "#d6d7e7"
+rootCommand:                         fbsetroot -solid "#d6d7e7"
 *font:                               glisp
diff -rNu fluxbox-0.9.14/data/styles.orig/Minimal fluxbox-0.9.14/data/styles/Minimal
--- fluxbox-0.9.14/data/styles.orig/Minimal	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Minimal	2006-02-08 13:44:01.000000000 +0100
@@ -76,4 +76,4 @@
 
 *Font:                          -*-helvetica-*-r-*-*-12-*-*-*-*-*-*-*
 
-rootCommand:                    bsetroot -solid SlateGrey
+rootCommand:                    fbsetroot -solid SlateGrey
diff -rNu fluxbox-0.9.14/data/styles.orig/Nyz fluxbox-0.9.14/data/styles/Nyz
--- fluxbox-0.9.14/data/styles.orig/Nyz	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Nyz	2006-02-08 13:44:01.000000000 +0100
@@ -100,4 +100,4 @@
 *textColor:                     grey85
 *Font:                          lucidasans-10
 
-rootCommand:                    bsetroot -solid grey20
+rootCommand:                    fbsetroot -solid grey20
diff -rNu fluxbox-0.9.14/data/styles.orig/Operation fluxbox-0.9.14/data/styles/Operation
--- fluxbox-0.9.14/data/styles.orig/Operation	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Operation	2006-02-08 13:44:01.000000000 +0100
@@ -83,7 +83,7 @@
 *textColor:                     grey85
 *Font:                          lucidasans-10
 
-rootCommand:                    bsetroot -gradient flatcrossdiagonalgradient -from rgb:4/6/8 -to rgb:3/48/6
+rootCommand:                    fbsetroot -gradient flatcrossdiagonalgradient -from rgb:4/6/8 -to rgb:3/48/6
 
 bbpager.desktop.focus:          flat interlaced crossdiagonal gradient
 bbpager.desktop.focus.color:    rgb:6/9/c
diff -rNu fluxbox-0.9.14/data/styles.orig/Outcomes fluxbox-0.9.14/data/styles/Outcomes
--- fluxbox-0.9.14/data/styles.orig/Outcomes	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Outcomes	2006-02-08 13:44:01.000000000 +0100
@@ -107,7 +107,7 @@
 bevelWidth:                     2
 handleWidth:                    4
 
-rootCommand:                    bsetroot -mod 4 4 -fg rgb:54/6/6 -bg grey20
+rootCommand:                    fbsetroot -mod 4 4 -fg rgb:54/6/6 -bg grey20
 
 ! for the bbtools
 menuFont:                       lucidasans-10
diff -rNu fluxbox-0.9.14/data/styles.orig/Rampage fluxbox-0.9.14/data/styles/Rampage
--- fluxbox-0.9.14/data/styles.orig/Rampage	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Rampage	2006-02-08 13:44:01.000000000 +0100
@@ -99,4 +99,4 @@
 *font:                           -b&h-lucida-medium-r-normal-*-*-100-*-*-p-*-iso8859-*
 *textColor:                      black
 
-rootCommand:                     bsetroot -mod 16 8 -bg rgb:0/8/8 -fg rgb:0/9/9
+rootCommand:                     fbsetroot -mod 16 8 -bg rgb:0/8/8 -fg rgb:0/9/9
diff -rNu fluxbox-0.9.14/data/styles.orig/Results fluxbox-0.9.14/data/styles/Results
--- fluxbox-0.9.14/data/styles.orig/Results	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Results	2006-02-08 13:44:01.000000000 +0100
@@ -107,7 +107,7 @@
 bevelWidth:                     2
 handleWidth:                    4
 
-rootCommand:                    bsetroot -mod 4 4 -fg rgb:6/6/5c -bg grey20
+rootCommand:                    fbsetroot -mod 4 4 -fg rgb:6/6/5c -bg grey20
 
 ! for the bbtools
 menuFont:                       lucidasans-10
diff -rNu fluxbox-0.9.14/data/styles.orig/Shade fluxbox-0.9.14/data/styles/Shade
--- fluxbox-0.9.14/data/styles.orig/Shade	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Shade	2006-02-08 13:44:01.000000000 +0100
@@ -88,4 +88,4 @@
 *textColor:                     grey20
 *Font:                          lucidasans-10
 
-rootCommand:                    bsetroot -solid rgb:4/4/4
+rootCommand:                    fbsetroot -solid rgb:4/4/4
diff -rNu fluxbox-0.9.14/data/styles.orig/Spiff fluxbox-0.9.14/data/styles/Spiff
--- fluxbox-0.9.14/data/styles.orig/Spiff	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Spiff	2006-02-08 13:44:01.000000000 +0100
@@ -104,6 +104,6 @@
 borderWidth:                    1
 handleWidth:                    3
 
-rootCommand:                    bsetroot -mod 2 2 -fg rgb:48/50/68 -bg rgb:50/58/70
+rootCommand:                    fbsetroot -mod 2 2 -fg rgb:48/50/68 -bg rgb:50/58/70
 
 *Font:  lucidasans-10
diff -rNu fluxbox-0.9.14/data/styles.orig/Twice fluxbox-0.9.14/data/styles/Twice
--- fluxbox-0.9.14/data/styles.orig/Twice	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/Twice	2006-02-08 13:44:01.000000000 +0100
@@ -100,4 +100,4 @@
 *textColor:                     grey85
 *Font:                          lucidasans-10
 
-rootCommand:                    bsetroot -solid grey20
+rootCommand:                    fbsetroot -solid grey20
diff -rNu fluxbox-0.9.14/data/styles.orig/qnx-photon fluxbox-0.9.14/data/styles/qnx-photon
--- fluxbox-0.9.14/data/styles.orig/qnx-photon	2006-02-08 13:41:01.000000000 +0100
+++ fluxbox-0.9.14/data/styles/qnx-photon	2006-02-08 13:44:01.000000000 +0100
@@ -131,4 +131,4 @@
 bevelWidth:                          2
 handleWidth:                         4
 frameWidth:                          0
-rootCommand:                         bsetroot -solid white
+rootCommand:                         fbsetroot -solid white
