diff -urN fluxbox-0.9.7.orig/nls/bg_BG/BaseDisplay.m fluxbox-0.9.7/nls/bg_BG/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/bg_BG/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/bg_BG/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=cp1251
+
 $ #XError
 # %s:  X „Â¯Í‡: %s(%d) opcodes %d/%d\n  resource 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/convert.awk fluxbox-0.9.7/nls/convert.awk
--- fluxbox-0.9.7.orig/nls/convert.awk	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/convert.awk	2004-01-05 21:42:01.000000000 +0100
@@ -29,6 +29,8 @@
   text = substr($0, 3)  
   printf "%d %s\n", minor, text > output
 }
-  
+
+/^\$ codeset/ { print > output }
+
 ! /^(\$|#)/ { print > output }
 
diff -urN fluxbox-0.9.7.orig/nls/da_DK/BaseDisplay.m fluxbox-0.9.7/nls/da_DK/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/da_DK/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/da_DK/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  X fejl: %s(%d) kode %d/%d\n  resource 0x%lx\n
 $ #SignalCaught
Pliki fluxbox-0.9.7.orig/nls/da_DK/.BaseDisplay.m.swp i fluxbox-0.9.7/nls/da_DK/.BaseDisplay.m.swp rÛøni± siÍ
diff -urN fluxbox-0.9.7.orig/nls/de_DE/BaseDisplay.m fluxbox-0.9.7/nls/de_DE/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/de_DE/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/de_DE/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  X Fehler ! : %s(%d) opcodes %d/%d\n  resource 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/es_ES/BaseDisplay.m fluxbox-0.9.7/nls/es_ES/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/es_ES/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/es_ES/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  Un error de X: %s(%d) clave de operaciÛn %d/%d\n  recurso 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/et_EE/BaseDisplay.m fluxbox-0.9.7/nls/et_EE/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/et_EE/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/et_EE/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  X'i viga: %s(%d) opkood %d/%d\n  resurss 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/fr_FR/BaseDisplay.m fluxbox-0.9.7/nls/fr_FR/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/fr_FR/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/fr_FR/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  Erreur de X %s(%d) opcodes %d/%d\n  ressource 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/it_IT/BaseDisplay.m fluxbox-0.9.7/nls/it_IT/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/it_IT/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/it_IT/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  errore di X: %s(%d) opcodes %d/%d\n  resource 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/ja_JP/BaseDisplay.m fluxbox-0.9.7/nls/ja_JP/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/ja_JP/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/ja_JP/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=eucjp
+
 $ #XError
 # %s:  X •®•È°º: %s(%d) opcodes %d/%d\n  resource 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/lv_LV/BaseDisplay.m fluxbox-0.9.7/nls/lv_LV/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/lv_LV/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/lv_LV/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-13
+
 $ #XError
 # %s:  X kÔ˚da: %s(%d) opkodi %d/%d\n  resurss 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/nl_NL/BaseDisplay.m fluxbox-0.9.7/nls/nl_NL/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/nl_NL/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/nl_NL/BaseDisplay.m	2004-01-05 21:44:29.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  X error: %s(%d) opcodes %d/%d\n  resource 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/pt_BR/BaseDisplay.m fluxbox-0.9.7/nls/pt_BR/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/pt_BR/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/pt_BR/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  X erro: %s(%d) opcodes %d/%d\n  recurso 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/pt_PT/BaseDisplay.m fluxbox-0.9.7/nls/pt_PT/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/pt_PT/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/pt_PT/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  X erro: %s(%d) opÁıes %d/%d\n  recurso 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/ru_RU/BaseDisplay.m fluxbox-0.9.7/nls/ru_RU/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/ru_RU/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/ru_RU/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-5
+
 $ #XError
 # %s:  œ€…¬À¡ X ”≈“◊≈“¡: %s(%d) ⁄Œ¡ﬁ≈Œ…— %d/%d\n “≈”’“” 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/sv_SE/BaseDisplay.m fluxbox-0.9.7/nls/sv_SE/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/sv_SE/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/sv_SE/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-1
+
 $ #XError
 # %s:  X-fel: %s(%d) opkod %d/%d\n  resurs 0x%lx\n
 $ #SignalCaught
diff -urN fluxbox-0.9.7.orig/nls/tr_TR/BaseDisplay.m fluxbox-0.9.7/nls/tr_TR/BaseDisplay.m
--- fluxbox-0.9.7.orig/nls/tr_TR/BaseDisplay.m	2004-01-05 21:45:59.000000000 +0100
+++ fluxbox-0.9.7/nls/tr_TR/BaseDisplay.m	2004-01-05 21:42:01.000000000 +0100
@@ -1,5 +1,7 @@
 $set 1 #BaseDisplay
 
+$ codeset=iso-8859-9
+
 $ #XError
 # %s : X hatas˝ : %s ( %d ) opkodlar %d/%d\n  kaynak 0x%lx\n
 $ #SignalCaught
