Only in fluxbox-0.1.13-bugfix1/data: menu.bak
diff -r -U 1 fluxbox-0.1.13/src/Ewmh.cc fluxbox-0.1.13-bugfix1/src/Ewmh.cc
--- fluxbox-0.1.13/src/Ewmh.cc	2002-10-17 01:32:17.000000000 +0200
+++ fluxbox-0.1.13-bugfix1/src/Ewmh.cc	2002-11-30 15:51:51.000000000 +0100
@@ -331,2 +331,5 @@
 	m_net_close_window = XInternAtom(disp, "_NET_CLOSE_WINDOW", False);
+	m_net_moveresize_window = XInternAtom(disp, "_NET_MOVERESIZE_WINDOW", False);
+	
+	// TODO: implement this one
 	m_net_wm_moveresize = XInternAtom(disp, "_NET_WM_MOVERESIZE", False);
diff -r -U 1 fluxbox-0.1.13/src/Font.cc fluxbox-0.1.13-bugfix1/src/Font.cc
--- fluxbox-0.1.13/src/Font.cc	2002-10-25 23:07:07.000000000 +0200
+++ fluxbox-0.1.13-bugfix1/src/Font.cc	2002-11-30 15:50:53.000000000 +0100
@@ -141,10 +141,4 @@
 		return false;
-	bool ret_val = m_fontimp->load(name);
-	if (ret_val && name == 0) { //prevent from having a bad fontimp
-		m_fontstr = name; // if the load really succeded then set font string
-	} else {
-		m_fontstr = "";
-	}
-
-	return ret_val;
+	m_fontstr = name;
+	return m_fontimp->load(name);
 }
diff -r -U 1 fluxbox-0.1.13/src/Screen.cc fluxbox-0.1.13-bugfix1/src/Screen.cc
--- fluxbox-0.1.13/src/Screen.cc	2002-11-17 12:27:50.000000000 +0100
+++ fluxbox-0.1.13-bugfix1/src/Screen.cc	2002-11-30 15:50:53.000000000 +0100
@@ -286,3 +286,3 @@
 			image_control, fluxbox->getStyleFilename(), getRootCommand().c_str());
-
+	theme->reconfigure(*resource.antialias);
 
@@ -449,3 +449,2 @@
 	XFlush(getBaseDisplay()->getXDisplay());
-	theme->reconfigure(*resource.antialias);
 }
diff -r -U 1 fluxbox-0.1.13/src/Texture.cc fluxbox-0.1.13-bugfix1/src/Texture.cc
--- fluxbox-0.1.13/src/Texture.cc	2002-09-19 21:51:55.000000000 +0200
+++ fluxbox-0.1.13-bugfix1/src/Texture.cc	2002-11-30 15:52:03.000000000 +0100
@@ -26,2 +26,6 @@
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif // HAVE_CONFIG_H
+
 #include "Texture.hh"
